webpack/InsightsCloudSync/Components/InsightsSettings/InsightsSettingsActions.js

```diff
@@ -19,7 +19,11 @@ export const getInsightsSyncSettings = () => async dispatch => {
         },
       },
     });
-  } catch ({ message }) {
+  } catch (err) {
+    const { message } = err;
+    if (err?.response?.status === 422) {
```

Comment:
Should this catch any non-2xx code? Any network connection can fail so it could return 5xx.

Reply:
Since I send myself the 422 specifically for this check, I thought it made sense to check for it specifically as well. But I may end up re-architecting this whole thing anyway, stand by..

Reply:
In that case, perhaps it should raise a more specific exception than `Error`?